dll/win32/devmgr/properties/advprop.cpp

```diff
@@ -1897,6 +1897,33 @@ AdvProcDetailsDlgProc(IN HWND hwndDlg,
     {
         switch (uMsg)
         {
+            case WM_CONTEXTMENU:
+            {
+                if ((HWND)wParam == GetDlgItem(hwndDlg, IDC_DETAILSPROPVALUE))
+                {
+                    WCHAR szColName[255];
+
+                    if (LoadStringW(hDllInstance, IDS_COPY, szColName, _countof(szColName)))
```

Comment:
Note (optional): you could reduce the indentation level by doing:
```c
                    if (!LoadStringW(hDllInstance, IDS_COPY, szColName, _countof(szColName)))
                        break;
```
then continue with the original code as done below.